inst/stan/epinowcast.stan

```diff
@@ -156,8 +159,8 @@ model {
   sqrt_phi ~ normal(sqrt_phi_p[1], sqrt_phi_p[2]) T[0,];
   // log density: observed vs model
   if (likelihood) {
-    target += reduce_sum(obs_lupmf, st, 1, obs, sl, imp_obs, sg, st, rdlurd,
-                         srdlh, ref_lh, dpmfs, ref_p, phi);
+    target += reduce_sum(obs_lupmf, st, 1, flat_obs, sl, csl, imp_obs, sg, st,
+                         rdlurd, srdlh, ref_lh, dpmfs, ref_p, phi);
```

Comment:
If I am not mistaken, doing it this way does not allow the scheduler to select slices with only parts of a reference date _x_ group observation (i.e. only parts of the corresponding row in the reporting triangle). I think this is fine (the big advantage of doing it this way is that we can have the selection from `sg`, `st` and `sl` directly using the indices between `start:end`). Just to keep in mind that we have a tiny bit of additional flexibility for the scheduler left in case we want to change this later.

Reply:
Yes, you are correct. Useful to flag.